pkg/daemon/ceph/client/image.go

```diff
@@ -141,6 +142,13 @@ func DeleteImageFromTrashInRadosNamespace(context *clusterd.Context, clusterInfo
 	return nil
 }
 
+// for a given size in bytes, round up to the nearest number of Mibibytes,
+// i. e. return the smallest  number of Mibibytes larger than or equal to size.
+func roundup_size_MiB(size uint64) uint64 {
```

Comment:
camelCase is standard for golang methods
```suggestion
func roundupSizeMiB(size uint64) uint64 {
```

Reply:
applied